TheMovieDb/Home/Utils/MovieCellProtocol.swift

```diff
@@ -0,0 +1,12 @@
+//
+//  MovieCellProtocol.swift
+//  TheMovieDb
+//
+//  Created by Javier Cueto on 30/10/21.
+//
+
+import UIKit
+
+protocol MovieCellProtocol: UICollectionViewCell {
+    func withMovie(with movie: Movie)
+}
```

Comment:
this is not bad, but a little unnecessary, because  we are type casting specifically each personalized cell

Reply:
I removed the protocol and now I'm usign a viewModel to pass data to each cell